logdna/resource_view_test.go

```diff
@@ -570,6 +570,7 @@ func TestViewMultipleChannels(t *testing.T) {
 func testViewConfigMultipleChannelsInvalidJSON() string {
 	return fmt.Sprintf(`provider "logdna" {
 		servicekey = "%s"
+		url = "http://api.use.stage.logdna.net"
```

Comment:
testing against staging for now- can remove these url's when alert presets goes live on Monday